.github/workflows/ci.yml

```
@@ -80,6 +80,6 @@ jobs:
 
             - name: 'Run SonarQube cloud scanner'
               if: ${{ matrix.php == env.LATEST_SUPPORTED_PHP_VERSION }}
-              uses: minvws/nl-irealisatie-generic-pipelines/.github/actions/sonarcloud@main
+              uses: minvws/action-sonarcube-cloud@update-action
```

I've tagged the [`v1.0.0` release](https://github.com/minvws/action-sonarcube-cloud/releases/tag/v1.0.0) as well as the [dynamic `v1` tag](https://github.com/minvws/action-sonarcube-cloud/releases/tag/v1).

```suggestion
              uses: minvws/action-sonarcube-cloud@v1
```